packages/dashboard/src/karma/apiProviderFixture.js

```diff
@@ -66,7 +66,17 @@ export default function ApiProviderFixture({ children }) {
   const usersApi = useMemo(
     () => ({
       getAuthors: () =>
-        Promise.resolve(formattedStoriesArray.map((story) => story.author)),
+        Promise.resolve(
+          Object.values(
+            formattedStoriesArray.reduce((dict, { author }) => {
+              if (dict[author.id] === undefined) {
+                dict[author.id] = author;
+              }
+
+              return dict;
+            }, {})
```

Comment:
Makes sure that authors are unique. Probably a prettier way to do this so lmk if you want a one line solution lol